src/Integration.UnitTests/Binding/CSharpVBBindingOperationTests.cs

```diff
@@ -364,13 +364,16 @@ public void ProjectBindingOperation_Commit_AddsAdditionalFile()
 
             // Assert
             projectMock.Files.ContainsKey(cSharpVBBindingConfig.AdditionalFile.Path).Should().BeTrue("Should add AdditionalFile to the project");
+
+            var fileProperties = projectMock.ProjectItemsMock[cSharpVBBindingConfig.AdditionalFile.Path].PropertiesMock;
+            fileProperties[Constants.ItemTypePropertyKey].Value.Should().Be(Constants.AdditionalFilesItemTypeName);
```

Comment:
Previously we couldn't check this because the ProjectItemsMock wasn't persisted.